defaults/main.yml

```diff
@@ -9,108 +9,221 @@
 #
 # .. include:: includes/all.rst
 
-# .. Roundcube source and deployment [[[1
+# .. Packages and installation [[[
+#
+# -----------------------------
+#   Packages and installation
+# -----------------------------
```

Comment:
You could convert the `defaults/main.yml` to the current standard by using the section headers with just one line underneath, and include `vim` fold markers. The whole file should also be enclosed in `vim` folds with a file header. See for example `debops.apt` role.